package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "lizardboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./bin/www"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Moniarchy/lizardboard.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Moniarchy/lizardboard/issues"
+  },
+  "homepage": "https://github.com/Moniarchy/lizardboard#readme",
+  "dependencies": {
+    "babel-cli": "^6.11.4",
```

Comment:
Please move all of the babel and testing (mocha) dependencies into the devDependencies section - we will not be using bable in production (the source will have been transpiled before being served in a production environment).